CMakeLists.txt

```diff
@@ -29,9 +29,18 @@ find_library(TOFI_COMPUTE
   NAMES tofi_compute
   PATHS /usr/lib)
 
+set(SOURCE ${CMAKE_SOURCE_DIR}/include/tof-ros2)
+
+set(CMAKE_INSTALL_PREFIX ${CMAKE_CURRENT_SOURCE_DIR}/../../tof_ros2_binary/install/)
+
 set(SOURCES ${CMAKE_SOURCE_DIR}/include/tof_ros2cpp)
+FILE(GLOB_RECURSE LibFiles ${CMAKE_SOURCE_DIR}/include/tof-ros2cpp/*.h)
```

Comment:
${CMAKE_SOURCE_DIR}/include/tof_ros2cpp/*.h
Wrong folder name use _ instead of -

Reply:
Thank you! 
Solved here: https://github.com/analogdevicesinc/tof-ros2/pull/3